tex/introduction.tex

```diff
@@ -15,7 +15,54 @@ \section{Introduction}
 
 \subsection{User requirements}
 
-(Might be formulated as user stories, both covering present and desired future workflows.)
+From a user perspective, notebooks may be attractive because they allow for combining a narrative with technical details, code, data and figures and hence provide an integrated way of presenting a line of scientific reasoning, and because they can be created in an incremental and interactive process that in some cases is close the the thought process during scientific work.
+
+Jupyter may be attractive for users because though originating from the Python ecosystem, it provides a common interface for creating notebooks that can contain code in many different languages.  Work with remote Jupyter instances, e.g., through a JupyterHub is popular, because the browser-based graphical application is executed on the user's computer and the bandwidth and stability requirements for the connection to the server are minimal and result in a low-latency and smooth user experience that is rare among classical GUIs that allow for execution on remote servers.
+
+Besides Jupyter, there are other popular implementations of computational notebooks~\citep{Hinsen2019a} that increasingly provide a browser-based way of working with them\footnote{See, e.g., the Matlab Live Editor at https://de.mathworks.com/products/matlab/live-editor.html.}.
+
+The following are user stories that may illustrate what people who end up working via a JuptyerHub on an HPC centre may have in mind:
+\begin{itemize}
+  \item I want to produce a computational notebook that presents my scientific data analysis.
+  All or some of my data are so big, that the processing needs substantial storage and computational resources.
+  \item I want to explore / analyze data that are on an HPC centre and that should not\footnote{e.g., forr reproducibility reasons}, or can not\footnote{e.g. because of volume} be moved to my computer.
+  For the exploration I need to try out and quickly adapt different ideas that I would like to document keeping code and results / figures closely together.
+  \item I am used to working with a specific notebook software and format that I mostly use on my personal computer.
+  Now I want to bring my expertise into a collaboration that runs large simulations on an HPC centre and I would like to keep working with my familiar software without having to learn much about the Unix command line etc.
+  \item I want to do data analysis from a personal device that I cannot use for the task directly, e.g., because I am not allowed to install software myself.
+  \item My device has a modern web browser installed that I can use to browse the internet.
+\end{itemize}
```

Comment:
Some of these things are very general and I would suggest to focus more on Jupyter@HPC aspects only? Because that is the problem we are trying to discuss/solve?

Reply:
Done.